From f8cd871a01a5e38e35cb4d44768dce09265265f4 Mon Sep 17 00:00:00 2001
From: adisbladis <adis@blad.is>
Date: Tue, 24 Oct 2017 02:00:45 +0800
Subject: [PATCH] Use ashift=12 by default on SSDSC2BW48 disks
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Currently the 480GB models of this disk do not use ashift=12 by
default.  SSDSC2BW48 is also optimized for 4k blocks.

Reviewed-by: Brian Behlendorf <behlendorf1@llnl.gov>
Reviewed-by: George Melikov <mail@gmelikov.ru>
Signed-off-by: adisbladis <adis@blad.is>
Closes #6774
---
 cmd/zpool/zpool_vdev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmd/zpool/zpool_vdev.c b/cmd/zpool/zpool_vdev.c
index 4c3793d6e..04a2611f9 100644
--- a/cmd/zpool/zpool_vdev.c
+++ b/cmd/zpool/zpool_vdev.c
@@ -188,6 +188,7 @@ static vdev_disk_db_entry_t vdev_disk_database[] = {
 	{"ATA     INTEL SSDSC2BB60", 4096},
 	{"ATA     INTEL SSDSC2BB80", 4096},
 	{"ATA     INTEL SSDSC2BW24", 4096},
+	{"ATA     INTEL SSDSC2BW48", 4096},
 	{"ATA     INTEL SSDSC2BP24", 4096},
 	{"ATA     INTEL SSDSC2BP48", 4096},
 	{"NA      SmrtStorSDLKAE9W", 4096},
-- 
2.15.1

